TeamCode/src/main/java/org/firstinspires/ftc/teamcode/BasicOpMode_Iterative.java

```diff
@@ -1,32 +1,3 @@
-/* Copyright (c) 2017 FIRST. All rights reserved.
```

Comment:
This license should remain. It is telling us what rights we have to modify and use the file.
To learn more about open source licenses, [click here](https://opensource.org/licenses).